src/tests/system/tests/test_logging.py

```diff
@@ -45,38 +50,91 @@ def test_logging__default_settings_logs_debug_level(client: Client):
+
+
+@pytest.mark.integration
+@pytest.mark.importance("low")
+@pytest.mark.topology(KnownTopology.Client)
+def test_logging__runtime_settings_logs_debug_level(client: Client):
+    """
+    :title: Domain log level from runtime is honored
+    :setup:
+        1. Configure SSSD for local system authentication
+        2. Clear logs and start SSSD with defaut level
+    :steps:
+        1. Check log file
+        2. Change domain debug level in runtime using sssctl debug-level
+        3. Check log file again
+    :expectedresults:
+        1. Logs messages contain default debug level 0x0070
+        2. Debug level change command succeeds
+        3. Logs messages contain default debug level 0x2f7f0
+    :customerscenario: False
+    """
+    client.sssd.common.local()
+    client.sssd.default_domain = "local"
+    client.sssd.clear(logs=True)
+    client.sssd.start(debug_level=None)
+    log_str = client.fs.read(client.sssd.logs.domain())
+    sssctl_set = client.host.conn.run("sssctl debug-level --domain local 9")
+    sssctl_get = client.host.conn.run("sssctl debug-level --domain local")
+    assert "level = 0x0070" in log_str, "Logs should contain debug level = 0x0070!"
+    assert sssctl_set.rc == 0, "'sssctl debug-level' command failed!"
+    assert sssctl_get.rc == 0, "'sssctl debug-level' command failed!"
+    assert " 0x2f7f0" in sssctl_get.stdout, "Sssctl debug-level output should contain 0x2f7f0"
+
+
+@pytest.mark.integration
+@pytest.mark.importance("low")
+@pytest.mark.topology(KnownTopologyGroup.AnyProvider)
+def test_logging__default_settings_does_not_log_user_logins(client: Client, provider: GenericProvider):
     """
     :title: Default debug level does not log user logins
     :setup:
         1. Create user
-        2. Configure SSSD for local system authentication
+        2. Configure SSSD for system authentication
         3. Clear cache and logs and start SSSD with default debug level
     :steps:
-        1. Store current logs and authenticate as a local user.
+        1. Store current logs and authenticate as an user.
         2. Compare stored logs with the current ones.
     :expectedresults:
         1. Login was successful
         2. Before event did not generate any new logs lines
     :customerscenario: False
     """
-
-    client.local.user("user1").add()
-    client.sssd.common.local()
-    client.sssd.default_domain = "local"
+    provider.user("user1").add(password="Secret123")
     client.sssd.domain["fallback_homedir"] = "/home/%%u"
+    client.sssd.sssd["services"] = "nss, pam, ssh"
 
     client.sssd.clear(logs=True, config=False)
     client.sssd.start(debug_level=None)
 
     client.fs.copy("/var/log/sssd", "/tmp/copy")
-    assert client.auth.ssh.password("user1", "Secret123"), "Login failed!"
+    assert client.auth.parametrize("ssh").password("user1", "Secret123"), "User failed login!"
     assert not client.host.conn.run("diff /var/log/sssd /tmp/copy").stdout, "Debug messages were generated!"
```

Comment:
This skips files that didn't exist at start:
```    
    Only in /var/log/sssd: gpo_child.log
    Only in /var/log/sssd: krb5_child.log
    Only in /var/log/sssd: sssd_kcm.log
```

Reply:
ack